tests/conftest.py

```diff
@@ -59,8 +59,8 @@ def sample_event() -> APIGatewayEvent:
     )
 
 
-@pytest.fixture(scope="session")
-def jwt_partial_payload() -> dict:
+@pytest.fixture(scope="session", name="jwt_partial_payload")
+def jwt_partial_authz() -> dict:
```

Comment:
Some time ago we agreed to add the `_fixture` suffix to avoid `pylint` warnings.

Changing the name of the fixture function is not recommended as you introduce a new name.